.github/actions/publish/action.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: build and draft GitHub release
+# https://github.com/product-os/flowzone/tree/master/.github/actions
+inputs:
+  json:
+    description: "JSON stringified object containing all the inputs from the calling workflow"
+    required: true
+  secrets:
+    description: "JSON stringified object containing all the secrets from the calling workflow"
+    required: true
+  variables:
+    description: "JSON stringified object containing all the variables from the calling workflow"
+    required: true
+
+runs:
+  # https://docs.github.com/en/actions/creating-actions/creating-a-composite-action
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        docker compose -f docker-compose.build.yml up build \
```

Comment:
@klutchell this is the best I could come up with given that our docker test steps don't preserve the workspace. Maybe there is a better way (or we preserve workspace in the docker flow as well as the images).

Reply:
> given that our docker test steps don't preserve the workspace

What do you mean by this?

Reply:
What I mean is that anything done in compose during testing isn't persisted in the workspace. For example, if the compose tests generate some build assets, these aren't passed to the publishing step. This is by design, since docker steps are concerned with images, not files in the workspace. However, in cases such as this, where we use the compose flow to build binaries, it adds a bit of friction.